be/src/common/config.h

```diff
@@ -903,4 +903,6 @@ CONF_Int32(binlog_page_max_size, "1048576");
 
 CONF_mInt64(txn_info_history_size, "20000");
 
+CONF_mInt64(load_tablet_limit_time, "30");
```

Comment:
load_tablet_limit_seconds is clearer.

Reply:
> load_tablet_limit_seconds is clearer.

done, changed in https://github.com/StarRocks/starrocks/pull/19347/commits/ab09d917afdd3617d180b83cb64bf9ae98ec8cd2